redaxo/src/core/pages/system.settings.php

```diff
@@ -88,12 +92,38 @@
         }
     }
 
-    foreach (rex_system_setting::getAll() as $setting) {
-        $key = $setting->getKey();
-        if (isset($settings[$key])) {
-            if (true !== ($msg = $setting->setValue($settings[$key]))) {
-                $error[] = $msg;
-            }
+    foreach ($settings as $key => $value) {
+        switch ($key) {
+            case 'start_article_id':
+            case 'notfound_article_id':
+                $value = (int) $value;
+                $article = Article::get($value);
+                if (!$article instanceof Article) {
+                    $error[] = I18n::msg('system_setting_' . $key . '_invalid');
+                }
+                Core::setConfig($key, $value);
+                break;
+
+            case 'default_template_id':
+                $value = (int) $value;
+                $sql = Sql::factory();
+                $sql->setQuery('SELECT * FROM ' . Core::getTablePrefix() . 'template WHERE id=? AND active=1', [$value]);
+                if (1 != $sql->getRows() && 0 != $value) {
+                    $error[] = I18n::msg('system_setting_default_template_id_invalid');
+                }
+                Core::setConfig('default_template_id', $value);
+                break;
+
+            case 'article_history':
+            case 'article_work_version':
+                $value = (bool) $value;
+                Core::setConfig($key, $value);
+                break;
+
+            case 'phpmailer_errormail':
+                $value = (int) $value;
+                Config::set('phpmailer', 'errormail', $value);
```

Comment:
Haben wir beim Auflösen des Addons anscheinend übersehen. Eigentlich haben wir die phpmailer-Config-Werte in den Core-Config-Namespace verschoben.
`Core::setConfig('phpmailer_errormail', $value);`

Kannst du die Stellen anpassen? Wegen mir ruhig hier mit im PR. Oder lieber separat?